src/applications/vaos/new-appointment/components/ReviewPage/index.jsx

```diff
@@ -95,33 +94,21 @@ export default function ReviewPage() {
               {submitStatusVaos400 ? (
                 <p>
                   We’re sorry. Something went wrong when we tried to submit your{' '}
-                  {submissionType}. You’ll need to call your local VA medical
-                  center to schedule this appointment.
+                  {submissionType}. Call your VA medical center to schedule this{' '}
+                  {submissionType}.
                 </p>
               ) : (
                 <p>
                   We’re sorry. Something went wrong when we tried to submit your{' '}
-                  {submissionType} and you’ll need to start over. We suggest you
-                  wait a day to try again or you can call your medical center to
-                  help with your {submissionType}.
+                  {submissionType}. You can try again later, or call your VA
+                  medical center to help with your {submissionType}.
                 </p>
               )}
               <>
-                {!facilityDetails && (
-                  <NewTabAnchor
-                    href={`/find-locations/facility/vha_${getRealFacilityId(
-                      data.vaFacility || data.communityCareSystemId,
-                    )}`}
-                  >
-                    {submitStatusVaos400
-                      ? 'Find facility contact information'
-                      : 'Contact your local VA medical center'}
-                  </NewTabAnchor>
-                )}
-                {!!facilityDetails && (
+                {(!!facilityDetails || parentFacility) && (
```

Comment:
Given that we're repeating the `|| ` failover a few times in this file, could be worth creating a variable up at the start of the function that does the fallback once?